substrate/runtime-sandbox/with_std.rs

```diff
@@ -404,4 +412,71 @@ mod tests {
 		).unwrap();
 		assert_eq!(return_val, ReturnValue::Value(TypedValue::I32(0x1337)));
 	}
+
+	#[test]
+	fn signatures_dont_matter() {
```

Comment:
This is the test that actually ensures that libsandbox actually doesn't care of signatures.